ERCS/erc-7715.md

```diff
@@ -318,7 +445,8 @@ Example:
   "sessionProperties": {
     "permissions": "true",
     "signerTypes": "keys,account",
-    "permissionTypes": "erc20-limit,contract-call"
+    "permissionTypes": "erc20-transfer,contract-call",
+    "policyTypes": "token-allowance,gas-limit"
```

Comment:
This should be `globalPolicyTypes`, since most policies will be properties of permissions, so it's enough to just declare support for the permission type (which by extension supports all the associated policies).  It's only global policies that will need explicit declarations of support.  So in this example, `gas-limit` would be a global policy but `token-allowance` wouldn't (since it's just a policy under `erc20-transfer`).

Reply:
but technically there would be a `global-gas-limit` and `gas-limit` policy separately

because you might want to set a `gas-limit` for a certain permission regarding swaps, minting, etc

also `token-allowance` is not global but it's also not exclusive to `erc20-transfer` as it can be used with other permissions such as `erc1155-transfer` and `native-transfer`